lib/memory_profiler/results.rb

```diff
@@ -99,7 +100,7 @@ def dump_strings(io, title, strings)
       io.puts "#{title} String Report"
       io.puts @colorize.line("-----------------------------------")
       strings.each do |string, stats|
-        io.puts "#{stats.reduce(0) { |a, b| a + b[1] }.to_s.rjust(10)}  #{@colorize.string((string[0..200].inspect))}"
+        io.puts "#{stats.reduce(0) { |a, b| a + b[1] }.to_s.rjust(10)}  #{@colorize.string((string[0,200].inspect))}"
```

Comment:
Of note, this is now the first commit, but this change was simply to match what I assume was the real intent of this line:  "Display the first 200 characters of the string"

The difference in the two lines:

* `string[0..200]` will return a substring between index `0` and index `200` (201 characters)
* `string[0,200]` will return a string starting at index `0` that is 200 characters long

The latter seems like proper original intent, and was necessary for the tests in the last commit to pass when testing the results of the original report to the `:mem_lite` version.